### DISCUSSION TITLE
Unable to send mail from Webmail

### DISCUSSION THREAD

**Discussion by fxp1303** [modoboa/modoboa#2172] [Help]
Getting 500 internal server error when attempting to send email from webmail component

Any assistance would be very much appreciated.

![Screen Shot 2021-02-11 at 9 34 11 am](https://user-images.githubusercontent.com/78726921/107583318-d31bde80-6c4e-11eb-88ec-2353e2465092.png)

**Comment by tonioo** [comment]
Can you set DEBUG to True inside settings.py, reload uwsgi and try again? Look at the response tab.

**Reply by fxp1303** [reply]
Upload screenshots to assist with debugging the issue, strange that response is connection refused...
![Screen Shot 2021-02-12 at 4 25 53 pm](https://user-images.githubusercontent.com/78726921/107733950-5c670a00-6d50-11eb-9409-2c17e1d44408.png)
![Screen Shot 2021-02-12 at 4 29 11 pm](https://user-images.githubusercontent.com/78726921/107733957-5ec96400-6d50-11eb-835a-0914a394f5a5.png)
![Screen Shot 2021-02-12 at 4 29 38 pm](https://user-images.githubusercontent.com/78726921/107733960-60932780-6d50-11eb-833d-8b19a047bdd7.png)
![Screen Shot 2021-02-12 at 4 28 51 pm](https://user-images.githubusercontent.com/78726921/107733963-61c45480-6d50-11eb-8f03-2581caf0f191.png)

**Reply by tonioo** [reply]
@fxp1303 Have you enabled DEBUG mode?

**Reply by fxp1303** [reply]
Yes and restart uwgi

**Reply by tonioo** [reply]
Have you checked mail logs? (/var/log/mail.log file)

**Reply by Sysadminfromhell** [reply]
I had the same problem, did you try to restart the server and check then the syslog? alternativ check mail.log or mail.err if this is configurered. Helpful would be also the access.log and error.log from the webmail website.

**Reply by fxp1303** [reply]
Yes, I have scanned all the logs and no indication where the issue is, although reading the 500 response in more details and reviewing the code plus more test with other parts of the webmail app and code....in my limited knowledge about the application there is an issue related to the follow line of code;

from .forms import (
    FolderForm, AttachmentForm, ComposeMailForm, ForwardMailForm
)

Because all of these form are not working AttachmentForm, ComposeMailForm, ForwardMailForm

Is it possible that the importing the .forms modules has not happened?

**Reply by tonioo** [reply]
I don't think so.
I looks like the webmail is not able to connect to the SMTP server when it sens the email. Have you checked online settings for the SMTP connection?
